app/src/composables/use-mutex.ts

```diff
@@ -0,0 +1,59 @@
+import { sleep } from '@directus/sdk';
```

Comment:
Feels wrong to import that from the SDK, should come from `/utils`, `@directus/utils` or be defined inline